.github/workflows/pull-requests.yaml

```
@@ -55,35 +56,186 @@ jobs:
+        run: |
+          cd ..
+          rm -rf /tmp/$SANDBOX_NAME
+          cp -r cozystack /tmp/$SANDBOX_NAME
+          sudo systemctl stop "rm-workspace-$SANDBOX_NAME.timer" "rm-workspace-$SANDBOX_NAME.service" 2>/dev/null || true
+          sudo systemctl reset-failed "rm-workspace-$SANDBOX_NAME.timer" "rm-workspace-$SANDBOX_NAME.service" 2>/dev/null || true
+          sudo systemctl daemon-reexec
+          sudo systemd-run \
+            --on-calendar="$(date -d 'now + 24 hours' '+%Y-%m-%d %H:%M:%S')" \
+            --unit=rm-workspace-$SANDBOX_NAME \
+            rm -rf /tmp/$SANDBOX_NAME
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)